supabase/migrations/20260817183727_user_onboarding_details_step_comment.sql

```
@@ -0,0 +1,2 @@
+COMMENT ON COLUMN "public"."users"."onboarding" IS
+  'Persisted create-app onboarding wizard progress for resume and admin drop-off. Keys: status, step, flow, intent, details_step, app_name, app_id, existing_app, existing_app_setup, store_url, imported_store_app_id, org_name, estimated_users_index, onboarding_attempt_id, last_run_id, updated_at, completed_at.';
```

SQLFluff (4.2.2)

[error] Line is too long (312 > 80).

(LT05)